rest/client/interface.go

```diff
@@ -126,4 +126,7 @@ type Communicator interface {
 
 	// Fetch the current authenticated user's public keys
 	GetCurrentUsersKeys(context.Context) ([]restmodel.APIPubKey, error)
+
+	// TODO: strings or restmodel.APIPubKey?
```

Comment:
the struct is probably preferable. 

having said that, the fact that the api models don't use standard strings, might make that option less appealing